persistence/src/main/resources/application.conf

```diff
@@ -16,4 +16,5 @@ akka.persistence.snapshot-store.local.dir = "target/snapshots"
 hmda {
   actor-lookup-timeout = 5
   persistent-actor-timeout = 60
+  actor-flow-parallelism = 4
```

Comment:
From the Akka Documentation:

`There is a performance advantage of using parallelism > 1 even though the actor will only process one message at a time because then there is already a message in the mailbox when the actor has completed previous message.`

Most examples I've seen use values from 2 to 5. What's important is that it's greater than 1. In any case I've made this configurable should we ever go down to this level of tuning